docs/migrations.md

```diff
@@ -31,7 +32,9 @@ into the concepts presented throughout this page:
 Any time you add or change a field on a Django model, Wagtail page model
 (which are a particular kind of Django model), or StreamField block classes, a
 [Django schema migration](https://docs.djangoproject.com/en/1.11/topics/migrations)
-will be required.
+will be required. 
+When that field is a StreamField, this includes changes to the properties of a child block,
+or even something minor like a help text string.
```

Comment:
The "or" here confused me on first read. Changes to something minor like a help text string require migrations for any Django model field, not just StreamFields. Maybe change to "child block, even for changes as small as a modified help text string."

Reply:
I support explicitly mentioning things like `help_text` that people might not expect to cause a need for a migration, but isn't the first part of this statement covered by the "or Streamfield block class" phrase above?

Reply:
I think removing the "or" works:

```
When that field is a StreamField, this includes changes to the properties of a child block,
even something minor like a help text string.
```

Reply:
@Scotchester it adds context for "child block". You could have a thrice-nested block and changing the `help_text` still requires a migration on a page that uses a block that uses a block that uses the block you changed. That's not necessarily obvious.

Reply:
Nevermind, I see what you're saying. Will make it a bit clearer.